pkgs/applications/virtualization/xen/generic.nix

```diff
@@ -4,20 +4,18 @@ config:
 # Xen
 , bison, bzip2, checkpolicy, dev86, figlet, flex, gettext, glib
 , acpica-tools, libaio, libiconv, libuuid, ncurses, openssl, perl
-, python2Packages
-# python2Packages.python
 , xz, yajl, zlib
+, pythonPackages
```

Comment:
Please use python3Packages.

Reply:
This `pythonPackages` variable should be an argument passed in from either `4.10.nix` (where it is set to `python2Packages`) or `4.15.nix` (where it  is set to `python3Packages`). Maybe I could rename the variable to make that more clear?

Reply:
Yeah, Maybe pythonXPackages?